<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>421</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>420</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>重新开始答题</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>420</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>开始答题</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>421</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="select_table"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
